workflows/sweep_simulate.snake

```diff
@@ -717,3 +763,40 @@ rule get_sw_stats:
     run:
         dump_results(input, output)
 """
+
+
+rule diploshic_files:
+   input: 
+       output_dir + '/simulated_data/sweeps/{middle}/sim_{chrom}_{left}_{right}.trees'
+   output:
+       output_dir + '/simulated_data/sweeps/{middle}/sim_{chrom}_{left}_{right}.diploshic.ancFile'
+   run:
+        write_diploshic_files(input[0], output[0])
+
+rule diploshic_fvs:
+   input: 
+       output_dir + '/simulated_data/sweeps/{middle}/sim_{chrom}_{left}_{right}.trees',
+       output_dir + '/simulated_data/sweeps/{middle}/sim_{chrom}_{left}_{right}.vcf',
+       rules.diploshic_files.output
+   output:
+       output_dir + '/simulated_data/sweeps/{middle}/sim_{chrom}_{left}_{right}.diploshic.fv'
+   run:
+        seq_len = tskit.load(input[0]).sequence_length
```

Comment:
this is wrong because we trim the buffer regions before outputting the VCF (starting in line 403).

Reply:
okay we can take care of this by dumping out the sequence length of the VCF when it is written